config/config.local.toml

```diff
@@ -109,3 +109,8 @@ URI = "zkevm-prover:50071"
 [BroadcastServer]
 Host = "0.0.0.0"
 Port = 61090
+
+[Metrics]
+Host = "0.0.0.0"
+Port = 9091
+Enabled = false
```

Comment:
Instead of having this enable flag, Why don't you just check if host and port cointain valid info?? In that case if fields are empty the metrics would be disabled

Reply:
Combined all three fields as `URI`.